src/app/workflow/workflow.component.ts

```diff
@@ -161,16 +161,16 @@ export class WorkflowComponent extends Entry<WorkflowVersion> implements AfterVi
     // Entry type is set in the container, workflow, and service routing files.
     this.entryType = this.sessionQuery.getValue().entryType;
     if (this.entryType === EntryType.BioWorkflow) {
-      this.validTabs = ['info', 'launch', 'versions', 'files', 'tools', 'dag'];
+      this.validTabs = ['info', 'launch', 'versions', 'files', 'tools', 'dag', 'metrics'];
       this.redirectToCanonicalURL('/' + myBioWorkflowsURLSegment);
     } else if (this.entryType === EntryType.Tool) {
       this.validTabs = ['info', 'launch', 'versions', 'files'];
```

Comment:
@kathy-t After my questions to you about metrics and tools in Slack regarding my ticket, should `metrics` be here as well?

Reply:
Hmm, thought this only affected legacy tools, but I guess app tools get these `EntryType.Tool` tabs as well :thinking: I suppose so, so that app tool metrics tab stays when refreshed (@hyunnaye does the metrics tab stay on refresh for app tools with this PR?)

Legacy tools don't have the metrics tab because it doesn't use the workflow component and legacy tools don't have the launch with feature.

Reply:
Just checked and looks like app tools had this problem without adding `metrics` on this line. Just added!